src/main/java/ch/naviqore/raptor/router/LabelPostprocessor.java

```diff
@@ -160,6 +162,9 @@ List<Connection> reconstructParetoOptimalSolutions(List<StopLabelsAndTimes.Label
                 throw new IllegalStateException("Unknown label type");
             }
 
+            LocalDateTime departureTime = LocalDateTime.ofEpochSecond(departureTimestamp, 0, ZoneOffset.UTC);
```

Comment:
The timezone is defined per agency in the GTFS. So it i not always UTC. But as far as I understand from the docs it is always the same time zone per feed.

> agency_timezone (Timezone | Required): Timezone where the transit agency is located. If multiple agencies are specified in the dataset, each must have the same agency_timezone.

Reply:
For this application it doesn't matter which Timezone you use in the conversion, as the UTC Timestamp is only handled internally and reconverted to LocalDateTime. The timestamps are never returned outside of the raptor package.

Reply:
Hmm ok, but why then choosing the origin of POSIX time and introduce the need for long? We could just set the minimum time in the builder as origin and stay with ints?